templates/models/hasMany.stub

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
in next rev, I want to add functions like:
public function attachModel($model)
public function attachModels($model)
public function detachModel($model)
public function detachModels($model)